bedrock/base/templates/includes/sticky-promo.html

```diff
@@ -0,0 +1,40 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set promo_referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=sticky-promo' %}
+
+<aside class="mzp-c-sticky-promo mzp-t-product-firefox mzp-t-dark">
```

Comment:
Seeing as this relies on JS, suggest adding a class of `hide-from-legacy-ie` to hide the content from older browsers that don't get first-class CSS/JS?

Reply:
How does the `hide-from-legacy-ie` class work? By moving the `aside` below the `{% block content %}`, does that resolve the FOUC?

Reply:
We have an IE specific stylesheet that will hide elements using this class name: https://github.com/mozilla/bedrock/blob/master/media/css/protocol/common-old-ie.scss#L48-L50

For placement you may want to consider making a block in the base template (e.g. `{% sticky_promo_block %}` from which individual pages can populate: https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/base-protocol.html#L93-L102

Reply:
Moved and added the `hide-from-legacy-ie` for good measure. I did not move it into a block for now.